### ISSUE TITLE
Missing Enchanced Edition Menu Screen?

### ISSUE THREAD

**Issue by VAKAR1AN** [elishacloud/Silent-Hill-2-Enhancements#439]
Hey guys!

I've had this issue twice now where I've followed the steps and installed all the required files for the Enhanced Edition, but when I start up the game I get the Konami Logo, Title Card with Angela, and then it takes me to the main menu.

But it's the Director's Cut menu? I played the Enhanced Edition a few months ago and I remember there being an "Enhanced Edition" main menu?

I've tried twice now with reinstalling the files from a clean slate but I always seem to have the same issue. Also whenever I boot up the game I have black bars on either side of my screen?

I'm using a copy of SH2 from myabandonware, it's the Restless Dreams version?

Any help would be greatly appreciated!

(Also I can't seem to attach my d3d8 log?)

Thanks!

**Comment by AeroWidescreen** [comment]
> Also I can't seem to attach my d3d8 log?

It would certainly help us if you could. What is the reason you can't? Does d3d8.log not exist? Is the log empty? You can also open it  with notepad and paste the information here, but attaching the file is far more preferable.

> But it's the Director's Cut menu? I played the Enhanced Edition a few months ago and I remember there being an "Enhanced Edition" main menu?

It sounds like you either didn't download and install the Image Enhancement Pack, or the Enhancement Module (d3d8.dll) isn't working correctly. Again, uploading your log file would help us a lot.

**Comment by VAKAR1AN** [comment]
Hi Aero, thanks for the response!

When I try to upload the d3db file it says that "We don't support that file type?" Do I need to convert it or something?

I've attatched what my SH2 file looks like

![Screenshot (2)](https://user-images.githubusercontent.com/86529969/123523280-8e9f5580-d6ba-11eb-9d83-04b0b43ff3ac.png)

**Comment by Polymega** [comment]
It looks like you are trying to upload the `d3d8.cfg` file whereas we need the `d3d8.log` file. I strongly recommend showing known file extension types on your PC; not just for us to help you troubleshoot but to do so in general--it's very useful to keep that on. 

You can show known file extensions by following these steps: https://www.wikihow.tech/Show-File-Extensions-on-Windows

**Comment by VAKAR1AN** [comment]
Ah, sorry about that! Thanks for the tip

I don't appear to have a .log version of the file at all?

![Screenshot (4)](https://user-images.githubusercontent.com/86529969/123524761-48e78a80-d6c4-11eb-89a5-72e0b54b5871.png)

**Comment by Polymega** [comment]
Hmmm... Upon running the game with our project it should produce a `d3d8.log` file in that directory.

I don't think this would have anything to do with it, but try moving the entire `Silent Hill 2` folder **out of** that `Silent Hill 2; Enhanced` folder. Perhaps the semicolon in that folder name is messing with our module?

**Comment by VAKAR1AN** [comment]
Oh my god...

Yep, removing that semi colon fixed all my issues! Game run's perfectly now!

You are a genius! Thank you so much for all the help!

**Comment by Polymega** [comment]
Great! Glad this was an easy solution. I'll close this ticket out then. Enjoy the game!

**Comment by elishacloud** [comment]
I ran a few tests on this issue on my computer.  I created the folder `D:\Games\Silent Hill 2 with;semicolen\Silent Hill 2` and tried to run the game from this folder.  However, Windows tries to load the module from `D:\Games\Silent Hill 2 with;semicolen\Silent Hill 2\semicolen\Silent Hill 2\d3d8.dll`, which is clearly wrong.

**This is clearly a Windows bug.**  No code from our module is running so there is no way we can fix this.

![image](https://user-images.githubusercontent.com/25625456/123571707-a7257380-d77f-11eb-9cd3-e6b08ba3a7a7.png)
